src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+public class Deadline extends Task {
+    private LocalDate end;
+
+    public Deadline(String description, LocalDate end) {
+        super(description);
+        this.end = end;
+    }
```

Comment:
Consider adding JavaDoc comments for this class and its public methods. I noticed that other classes in the task package also seem to lack JavaDoc comments for the necessary classes and methods, so that is something to take note of!

Reply:
Thank you for pointing it out! I will add the JavaDoc comments for all task classes later.